ui-backend/catalog-ui-splitter/pom.xml

```diff
@@ -43,12 +43,6 @@
                 <configuration>
                     <instructions>
                         <Bundle-SymbolicName>${project.artifactId}</Bundle-SymbolicName>
-                        <Embed-Dependency>
-                            platform-util
```

Comment:
Is this the last embed of platform-util in the repo?

Reply:
There are others i think, i can remove them